src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -284,11 +277,13 @@ private Request make(final byte[] boundary) throws IOException {
          * Copy until boundary reached.
          * @param target Output file channel
          * @param boundary Boundary
+         * @param body Origin request body
          * @throws IOException If fails
          * @checkstyle ExecutableStatementCountCheck (2 lines)
          */
         private void copy(final WritableByteChannel target,
-            final byte[] boundary) throws IOException {
+            final byte[] boundary,
+            final ReadableByteChannel body) throws IOException {
```

Comment:
@dalifreire you can put `body` in the previous line, and leave only throws in this one